src/js/components/LogItem/LogItem.jsx

```diff
@@ -341,133 +351,171 @@ const LogItem = ({
 
                             {type === LOG_TYPE_FRAME && (
                                 <>
-                                    <TypographyBold>frame type</TypographyBold>
-                                    <TypographyMono>
-                                        {`${
-                                            frameNamesByType[frameParameters.type]
-                                        } (0x${
-                                            utils.getHexFromNumber(frameParameters.type, {separator: ''})
-                                        })`}
-                                    </TypographyMono>
-                                    <TypographyBold>access level</TypographyBold>
-                                    <TypographyMono>
-                                        {`${
-                                            accessLevelNames[frameParameters.accessLevel]
-                                        } (0x${
-                                            utils.getHexFromNumber(frameParameters.accessLevel, {separator: ''})
-                                        })`}
-                                    </TypographyMono>
-                                    <TypographyBold>destination address</TypographyBold>
-                                    <TypographyMono>
-                                        {`0x${utils.getHexFromNumber(frameParameters.destination, {separator: ''})}`}
-                                    </TypographyMono>
-                                    <TypographyBold>source address</TypographyBold>
-                                    <TypographyMono>
-                                        {`0x${utils.getHexFromNumber(frameParameters.source, {separator: ''})}`}
-                                    </TypographyMono>
-                                    <TypographyBold>message ID</TypographyBold>
-                                    <TypographyMono>{frameParameters.messageId}</TypographyMono>
+                                    {!isNullOrUndefined(frameParameters.type) && (
```

Comment:
```suggestion
                                    {!!(frameParameters.type) && (
```

?

Reply:
I removed the function where the value cannot be 0.